kore/src/main/scala/org/kframework/definition/outer.scala

```diff
@@ -97,6 +98,17 @@ case class Module(val name: String, val imports: Set[Module], localSentences: Se
       .groupBy(_.sort)
       .map { case (l, ps) => (l, ps) }
 
+  lazy val layouts: Set[String] =
+    productionsForSort
+      .get(Sorts.Layout)
+      .getOrElse(Set[Production]())
+      .collect({
+          case Production(_, Seq(RegexTerminal(_, terminalRegex, _)), _) => terminalRegex
+          case p => throw KEMException.compilerError("Productions of sort `Layout` must be exactly one `RegexTerminal`.\nProduction: " + p.toString())
```

Comment:
Should be #Layout

Reply:
It is `#Layout` defined as `val Layout = Sort("#Layout")` in `Sorts.scala`. Are you saying to also change the Scala/Java identifier name?

Reply:
Oh is this some kind of fancy string interpolation in scala I'm not aware of?

Reply:
No, I don't think so. It's just a constant called `Layout` defined in `Sorts.scala`, and that constant has value `Sort("#Layout")`.